Time zone issue in ExpiringKeystoreHealthCheckTest causes failures

It is about 7:30pm EST on Mar 19, 2021 and I was just running the tests. Three in `ExpiringKeystoreHealthCheckTest` unexpectedly failed:

* ShouldReportHealthy / whenThereAreOnlyValidCerts
* ShouldReportUnhealthy / whenThereAreExpiredCerts
* ShouldReportUnhealthy / whenThereAreExpiringCerts

All three have multiple failures on `issuedOn` and `expiresOn` dates (corresponding to `notBefore` and `notAfter` in `X509Certificate`, respectively).

Since they're all basically the same problem, here's the output from the `whenThereAreOnlyValidCerts` test for the `issuedOn` check:

```
java.lang.AssertionError: 
Expecting:
  "Tue, 19 Jan 2021 00:26:10 GMT"
to start with:
  "Mon, 18 Jan 2021"
```

and for the `expiresOn` check:

```
java.lang.AssertionError: 
Expecting:
  "Sun, 11 Sep 2022 00:26:10 GMT"
to start with:
  "Sat, 10 Sep 2022"
```

It certainly appears we have a problem with time zones, though I am not sure if this is limited only to the tests or if there is an actual problem with the code. Since running the tests when local time and UTC are on the same day works fine, I am _speculating_ it is a problem with the tests. But don't quote me on that.

--- a/src/test/java/org/kiwiproject/dropwizard/util/health/keystore/ExpiringKeystoreHealthCheckTest.java
+++ b/src/test/java/org/kiwiproject/dropwizard/util/health/keystore/ExpiringKeystoreHealthCheckTest.java
@@ -74,7 +74,7 @@ class ExpiringKeystoreHealthCheckTest {
             softly.assertThat(certInfo.getSubjectDN()).isEqualTo("CN=Unit Test, OU=Development, O=Project, L=Here, ST=VA, C=US");
             softly.assertThat(certInfo.getIssuerDN()).isEqualTo("CN=Unit Test, OU=Development, O=Project, L=Here, ST=VA, C=US");
 
-            var sixtyDaysAgo = ZonedDateTime.now(ZoneId.of("UTC")).minusDays(60);
+            var sixtyDaysAgo = ZonedDateTime.now().minusDays(60).toInstant().atZone(ZoneId.of("UTC"));
             var sixtyDaysAgoString = DATE_FORMATTER.format(sixtyDaysAgo);
             softly.assertThat(certInfo.getIssuedOn()).startsWith(sixtyDaysAgoString);
 
@@ -124,7 +124,7 @@ class ExpiringKeystoreHealthCheckTest {
             softly.assertThat(certInfo.getSubjectDN()).isEqualTo("CN=Unit Test, OU=Development, O=Project, L=Here, ST=VA, C=US");
             softly.assertThat(certInfo.getIssuerDN()).isEqualTo("CN=Unit Test, OU=Development, O=Project, L=Here, ST=VA, C=US");
 
-            var sixtyDaysAgo = ZonedDateTime.now(ZoneId.of("UTC")).minusDays(60);
+            var sixtyDaysAgo = ZonedDateTime.now().minusDays(60).toInstant().atZone(ZoneId.of("UTC"));
             var sixtyDaysAgoString = DATE_FORMATTER.format(sixtyDaysAgo);
             softly.assertThat(certInfo.getIssuedOn()).startsWith(sixtyDaysAgoString);
 
@@ -207,7 +207,7 @@ class ExpiringKeystoreHealthCheckTest {
             softly.assertThat(certInfo.getSubjectDN()).isEqualTo("CN=Valid Test, OU=Development, O=Project, L=There, ST=VA, C=US");
             softly.assertThat(certInfo.getIssuerDN()).isEqualTo("CN=Valid Test, OU=Development, O=Project, L=There, ST=VA, C=US");
 
-            var sixtyDaysAgo = ZonedDateTime.now(ZoneId.of("UTC")).minusDays(60);
+            var sixtyDaysAgo = ZonedDateTime.now().minusDays(60).toInstant().atZone(ZoneId.of("UTC"));
             var sixtyDaysAgoString = DATE_FORMATTER.format(sixtyDaysAgo);
             softly.assertThat(certInfo.getIssuedOn()).startsWith(sixtyDaysAgoString);
 

